lib/quby/questionnaires/entities/questions/date_question.rb

```diff
@@ -28,8 +28,8 @@ def day_key
             (@day_key || "#{key}_dd").to_s
           end
 
-          def input_keys
-            [day_key.to_sym, month_key.to_sym, year_key.to_sym]
+          def claimed_keys
```

Comment:
input_keys is nu weg, gaat dat wel goed met het default gedrag waar input_keys [key] is? De eigenlijke keys die aan de inputs hangen zijn [day_key.to_sym, month_key.to_sym, year_key.to_sym] toch